probe/appclient/app_client.go

```diff
@@ -192,6 +192,9 @@ func (c *appClient) Details() (xfer.Details, error) {
 	if err != nil {
 		return result, err
 	}
+	if resp.StatusCode != http.StatusOK {
+		return result, fmt.Errorf("Error response from %s: %s", c.url("/api"), resp.Status)
```

Comment:
Nit: based on your own log output (`Error response from http://172.16.0.3:4040//api: 404 Not Found`), maybe there's an extra `/` here:

```suggestion
		return result, fmt.Errorf("Error response from %s: %s", c.url("api"), resp.Status)
```

Reply:
The output is correct: the reason it's failing is the url has too many slashes in it.